### PR TITLE
Fixes #12572: node properties from inventory are not well loaded when executing queries on the web Interface

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/12572

This PR: 

- change the way the backend is running for NodeInfo related properties (see description in ticket), 
- remove the save of custom inventory properties from rudder nodes

The part where JsonSelect via JSONPath is done is added in PR: https://github.com/Normation/rudder/pull/1961

### CODE REVIEW

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

File: rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala [hunk 1]
```diff
@@ -468,22 +465,32 @@ trait NodeInfoServiceCached extends NodeInfoService with Loggable with CachedRep
...
+    def comp(a: Boolean, b: Boolean) = composition match {
+        case And => a && b
+        case Or  => a || b
+    }
+    // combine predicats according to comp
+    def combine(a: NodeInfoMatcher, b: NodeInfoMatcher) = new NodeInfoMatcher {
+      override def matches(node: NodeInfo): Boolean = comp(a.matches(node), b.matches(node))
+    }
+
+    val p = (if(predicats.isEmpty) {
```
Comment (VinceMacBuche) [inline rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala]: Maybe a comment to explain how we treat empty case and why it's true for And and false for Or

I understand it now, but will future ourselves will understand it ?

**Comment by fanf** [comment]
PR rebased

**Pull request merged**